src/reflect/scala/reflect/internal/Flags.scala

```diff
@@ -195,18 +195,21 @@ class Flags extends ModifierFlags {
   // The flags (1L << 59) to (1L << 63) are currently unused. If added to the InitialFlags mask,
   // they could be used as normal flags.
 
-  final val InitialFlags  = 0x0007FFFFFFFFFFFFL // normal flags, enabled from the first phase: 1L to (1L << 50)
+  final val InitialFlags  = 0x8007FFFFFFFFFFFFL // normal flags, enabled from the first phase: 1L to (1L << 50) and universe (63)
   final val LateFlags     = 0x00F8000000000000L // flags that override flags in (1L << 4) to (1L << 8): DEFERRED, FINAL, INTERFACE, METHOD, MODULE
   final val AntiFlags     = 0x0700000000000000L // flags that cancel flags in 1L to (1L << 2): PROTECTED, OVERRIDE, PRIVATE
   final val LateShift     = 47
   final val AntiShift     = 56
+  final val InCompilerUniverse = 0x8000000000000000L
+
+  final val PhaseIndependentFlags = 0x8007FFFFFFFFFE08L //(InitialFlags & ~(LateFlags >>> LateShift) & ~(AntiFlags >>> AntiShift)) | InCompilerUniverse
```

Comment:
I'd prefer to have the comment as code, assuming you can get the constant folder to play along, as I did in #6445. An alternative would be to add an `assert(0x8007FFFFFFFFFE08L == .... `) below.

Reply:
`InCompilerUniverse` can be removed if the overriding idea works out.